packages/support/lib/fs.js

```diff
@@ -36,21 +37,48 @@ const findRootCached = _.memoize(pkgDir.sync);
 const fs = {
   /**
    * Resolves `true` if `path` is _readable_, which differs from Node.js' default behavior of "can we see it?"
-   * @param {import('fs').PathLike} path
+   *
+   * On Windows, ACLs are not supported, so we only check for file existence.
+   * @param {PathLike} path
    * @returns {Promise<boolean>}
    */
   async hasAccess(path) {
     try {
-      await fsPromises.access(path, constants.R_OK);
+      if (isWindows()) {
+        await fsPromises.stat(path);
+      } else {
+        await fsPromises.access(path, constants.R_OK);
+      }
     } catch (err) {
       return false;
     }
     return true;
   },
 
+  /**
+   * Resolves `true` if `path` is executable; `false` otherwise.
+   *
+   * On Windows, ACLs are not supported; resolves `true` if `path` exists.
+   *
+   * This function will _not_ reject if the file does not exist.
+   * @param {PathLike} path
+   * @returns {Promise<boolean>}
+   */
+  async isExecutable(path) {
+    if (isWindows()) {
+      return fs.hasAccess(path);
```

Comment:
await ?

Reply:
hmm.. eslint didn't complain.   i'll fix it